content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,280 @@
+- name: PodSecurity
+  configuration:
+    apiVersion: pod-security.admission.config.k8s.io/v1beta1
+    kind: PodSecurityConfiguration
+    defaults:
+      enforce: "baseline"
+      enforce-version: "latest"
+      audit: "restricted"
+      audit-version: "latest"
+      warn: "restricted"
+      warn-version: "latest"
+    exemptions:
+      usernames: []
+      runtimeClasses: []
+      namespaces: [kube-system]
+EOF
+```
+
+# Configure the API server
+
+Let's configure API server to consume this file during cluster creation:
+
+```
+$ cat <<EOF > /tmp/pss/cluster-config.yaml 
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    apiServer:
+        extraArgs:
+          admission-control-config-file: /etc/config/cluster-level-pss.yaml
+        extraVolumes:
+          - name: accf
+            hostPath: /etc/config
+            mountPath: /etc/config
+            readOnly: false
+            pathType: "DirectoryOrCreate"
+  extraMounts:
+  - hostPath: /tmp/pss
+    containerPath: /etc/config
+    # optional: if set, the mount is read-only.
+    # default false
+    readOnly: false
+    # optional: if set, the mount needs SELinux relabeling.
+    # default false
+    selinuxRelabel: false
+    # optional: set propagation mode (None, HostToContainer or Bidirectional)
+    # see https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation
+    # default None
+    propagation: None
+EOF
+```
+
+_Note_: Please ensure that when using Docker Desktop with KinD, the `/tmp`
+directory is added as a Shared Directory under
+`Preferences -> Resources -> File Sharing` (as seen on Docker Desktop for Mac)
+
+# Create cluster with Pod Security Standards
+
+Now we are ready to create a cluster that consumes these pod security standards
+configured with pod security admission controller:
+
+```
+$ kind create cluster --name psa-with-cluster-pss --image kindest/node:latest --config /tmp/pss/cluster-config.yaml
+Creating cluster "psa-with-cluster-pss" ...
+ ✓ Ensuring node image (kindest/node:latest) 🖼 
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-psa-with-cluster-pss"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-psa-with-cluster-pss
+
+Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+
+
+$ kubectl cluster-info --context kind-psa-with-cluster-pss
+Kubernetes control plane is running at https://127.0.0.1:63855
+CoreDNS is running at https://127.0.0.1:63855/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+# Create pod
+
+Creating a pod with minimal configuration in default namespace should now be
+successful with warnings for `restricted` pod security standards:
```

Comment:
```suggestion
1. Create a Pod with minimal configuration in the default namespace:
```

Would this not have been possible before? We want to at least say what would be different in the original, non-PSA cluster.